tests/test_core.py

```diff
@@ -132,8 +226,16 @@ class TestMultiTrack(core.MultiTrack):
         def __init__(self, mtrack_id, data_home):
             self.mtrack_id = mtrack_id
             self._data_home = data_home
-            self.tracks = {t: TestTrack(t) for t in ["a", "b", "c"]}
-            self.track_audio_property = "f"
+            self._dataset_name = "foo"
+            self._index = None
+            self._metadata = None
+            self._track_class = TestTrack
+            self.track_ids = ["a", "b", "c"]
```

Comment:
now that you simplified the multitrack class, i think you can remove this init from the tests, and just pass in a simple index like `{'multitracks': {'mtrack_id': {'tracks': ['a', b', 'c']}}}`

I wrote similar tests for Track in #468